src/main/java/org/threadly/litesockets/SocketExecuterCommonBase.java

```diff
@@ -24,28 +25,28 @@
 /**
  *  This is a common base class for the Threaded and NoThread SocketExecuters. 
  */
-abstract class SocketExecuterCommonBase extends AbstractService implements SocketExecuterInterface {
+abstract class SocketExecuterCommonBase extends AbstractService implements SocketExecuter {
   public static final int WATCHDOG_CLEANUP_TIME = 30000;
-  protected final SchedulerServiceInterface acceptScheduler;
-  protected final SchedulerServiceInterface readScheduler;
-  protected final SchedulerServiceInterface writeScheduler;
-  protected final SimpleSchedulerInterface schedulerPool;
+  protected final SubmitterScheduler acceptScheduler;
+  protected final SubmitterScheduler readScheduler;
+  protected final SubmitterScheduler writeScheduler;
+  protected final SubmitterScheduler schedulerPool;
   protected final ConcurrentHashMap<SocketChannel, Client> clients = new ConcurrentHashMap<SocketChannel, Client>();
   protected final ConcurrentHashMap<SelectableChannel, Server> servers = new ConcurrentHashMap<SelectableChannel, Server>();
   protected final SocketExecuterByteStats stats = new SocketExecuterByteStats();
   protected final WatchdogCache dogCache;
-  protected Selector readSelector;
+  public Selector readSelector;
```

Comment:
it definitely shouldn't be, probably left over from some testing.  I will fix it